README.md

```diff
@@ -1,33 +1,189 @@
-# Operate First template for repositories
+# opfcli
```

Comment:
Can you please also include any dev docs or makefile in here? Like a base line for it - how to run `go build` and `go get`, etc.

Reply:
I've added a note about running `go build` to the README.